spinta/exceptions.py

```diff
@@ -878,7 +860,7 @@ class DatasetNameMissmatch(UserError):
 
 
 class DatasetSchemaRequiresIds(UserError):
-    template = "All given schema rows require UUID to be set in the id field."
+    template = "All given schema ({this}) rows require UUID to be set in the id field."
```

Comment:
I guess this will be a whole formatted json, which would just be reformatted badly, I suggest making a multi-line error here:

```suggestion
    template = ```
         All given schema rows require UUID to be set in the id field.
         Schema: {this}
   ```
```